### DISCUSSION TITLE
Wild Cards and dynamic prompts

### DISCUSSION THREAD

**Discussion by Kaleidia** [lllyasviel/Fooocus#223] [Ideas]
Hi, loving this software ;)
one thing i use a lot in a1111 is the dynamic prompt feature and the associated wild cards.

Is there a plan to add a feature like that in the future?

**Comment by GalaxyTimeMachine** [comment]
Wildcards are already supported, but I'm trying to find a way to get dynamic prompt to work.
eg. randomly choose one of these {one|two|three}

**Reply by StanleyPain** [reply]
Really stoked that wildcards are working. Would be cool to see dynamic prompts working at some point in time :).

As a workaround I've just been making wildcards in place. Takes a bit longer but works in a similar way.
